demostore_automation/tests/backend/products/product_reviews/test_product_reviews_smoke.py

```diff
@@ -34,7 +49,7 @@ def test_verify_create_product_review_endpoint_creates_review(self):
             review = generate_random_string(30)
             reviewer_name = generate_random_string(15)
             reviewer_email = generate_random_email_and_password()['email']
-            rating = random.randint(0, 5)
+            rating = random.randint(1, 5)
```

Comment:
User is unable to leave a 0 star review on website and rating count does not count 0 star reviews. Now the bug consistently shows 1 less than expected.